subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/serialization/codecs/Codecs.kt

```diff
@@ -178,6 +180,8 @@ class Codecs(
         bind(ownerService<BuildRequestMetaData>())
         bind(ownerService<WorkerExecutor>())
         bind(ownerService<ListenerManager>())
+        bind(ownerService<JavaInstallationProbe>())
```

Comment:
I wouldn't expose these services, instead we should calculate the JVM details on every build. The calculation can still continue to use these internal services, but not in a way that requires the references to these services to be serialised.